utils/bin/sre-login

```diff
@@ -1,6 +1,13 @@
 #!/bin/bash -e
 # OCM_CONTAINER_DOC: Logs into a cluster via the command line
 
+# Check that backplane is installed
+if ! ocm backplane version
```

Comment:
We'd probably want to pipe the output of this to /dev/null, right?

Reply:
Possibly. It seems to work okay without it though.
It may be handy seeing the error message in case something else is going wrong.

Reply:
Piped to dev null, thanks!